scripts/qa_crowbarsetup.sh

```diff
@@ -4922,6 +4922,9 @@ zypper -non-interactive --gpg-auto-import-keys --no-gpg-checks install ses-upgra
     for node in $ceph_mons; do
         ssh $node "ceph osd crush tunables firefly; ceph osd set require_jewel_osds"
     done
+
+    # wait for ceph cluser to recover after the upgrade
+    sleep 60
```

Comment:
Would be better to have a very short `wait_for` here so we can be sure that the `HEALTH_WARN` its only a temp thing due to the upgrade.

Im not sure about the exact timeframe here though, I guess it depends on the number of PGs?